website/content/docs/security/acl/acl-federated-datacenters.mdx

```diff
@@ -0,0 +1,232 @@
+description: >-
+  This topic describes the specific ACL bootstrapping policies that are necessary when ACLs are enabled for federated, multi-datacenter deployments.
+---
+
+# ACLs in Federated Datacenters
+
+This topic describes how to set up Consul's access control list (ACL) system
+in cluster deployments that span multiple data centers. This documentation is applicable
+to new clusters rather than existing clusters.
+
+-> **1.4.0 and later:** This guide only applies in Consul versions 1.4.0 and later.
+
+## Configure ACLs in the Primary Datacenter
+
+In a [federated Consul deployment](https://www.consul.io/docs/k8s/installation/multi-cluster), one of the datacenters is marked as the primary datacenter.
+The `acl` configuration block should be added to the primary datacenter server's configuration file
+as shown in the following example.
+
+See the [ACL Config Stanza](https://www.consul.io/docs/agent/options#acl) for more detailed descriptions of each option.
+
+-> **Versions before 1.11.0:** The `initial_management` token was called the `master` token in versions
+prior to 1.11.0
+
+<CodeTabs heading="ACL Configuration in Primary">
+
+```hcl
+bootstrap_expect =  3
+primary_datacenter =  "PRIMARY_DATACENTER_VALUE"
+acl = {
+ enabled =  true
+ default_policy = "deny"
+ down_policy = "deny"
+ enable_token_persistence = true
+ enable_token_replication = true
+ tokens = {
+   initial_management =  "ACL_MANAGEMENT_TOKEN"
+   agent = "YOUR_ACL_AGENT_TOKEN"
+  }
+}
+
+```
+
+```json
+{
+  "bootstrap_expect": N,
+  "primary_datacenter": "PRIMARY_DATACENTER_VALUE",
+  "acl": {
+    "enabled": true,
+    "default_policy": "deny",
+    "down_policy": "deny",
+    "enable_token_persistence": true,
+    "enable_token_replication": true,
+    "tokens": {
+      "initial_management": "ACL_MANAGEMENT_TOKEN",
+      "agent": "ACL_AGENT_TOKEN"
+    }
+  }
+}
+```
+
+</CodeTabs>
+
+~> **Warning:** Note that most enterprise deployments have security requirements that prevent specifying tokens in configuration files.
+The `enable_token_persistence` flag is also set in the configuration example so that the token is stored to disk in the agent's
+`data` directory. Any future changes to the token that are made through the [API](/api/agent#update-acl-tokens) will
+be persisted to the same location, and the value in the config file will be ignored.
+
+The ACL agent token can also be set using the CLI as shown below.
+
+```shell-session
+$ consul acl set-agent-token agent "<agent token here>"
+```
+
+## Configure Servers in Secondary Datacenters
+
+Servers in secondary data centers must be configured to point to the primary data center
+as shown in the following example. Secondary data centers also need the ACL replication token
+provided to them.
+
+### Create the replication token for ACL Management
+
+Replication tokens are needed for ACL token replication and
+to create both [local tokens](/api/acl/tokens#local) and [auth methods](/docs/acl/auth-methods)
```

Comment:
```suggestion
to create both [configuration entries](/docs/agent/config-entries) and [auth methods](/docs/acl/auth-methods)
```
Replication does not need to be enabled in order to create local tokens in a secondary datacenter. Suggest replacing with config entries instead.